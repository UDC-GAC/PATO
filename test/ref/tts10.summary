# Duplex-ID	TTSs (abs)	TTSs (rel)
ENSLCOT00000000041.1	33	0.000177
ENSLCOT00000000044.1	33	0.000183
ENSLCOT00000000056.1	33	0.00019
ENSLCOT00000000007.1	198	0.00294
ENSLCOT00000000004.1	1	5.77e-05
ENSLCOT00000000006.1	5	0.000132
ENSLCOT00000000009.1	1	2.47e-05
ENSLCOT00000000445.1	43	0.00057
ENSLCOT00000000484.1	43	0.000584
ENSLCOT00000000489.1	43	0.000585
ENSLCOT00000000017.1	4	0.000174
ENSLCOT00000000881.1	11	0.00011
ENSLCOT00000000921.1	11	0.000134
ENSLCOT00000000946.1	2	3.22e-05
ENSLCOT00000000013.1	32	0.00141
ENSLCOT00000000012.1	179	0.00246
ENSLCOT00000000078.1	3	3.06e-05
ENSLCOT00000000070.1	1	3.23e-05
ENSLCOT00000000131.1	29	0.000864
ENSLCOT00000000069.1	31	0.000845
ENSLCOT00000000043.1	126	0.0106
ENSLCOT00000000016.1	89	0.00182
ENSLCOT00000000323.1	208	0.0027
ENSLCOT00000000346.1	208	0.00299
ENSLCOT00000000018.1	10	0.000287
ENSLCOT00000000035.1	6	0.000141
ENSLCOT00000000014.1	213	0.00182
ENSLCOT00000000034.1	11	0.000532
ENSLCOT00000000251.1	181	0.000784
ENSLCOT00000000255.1	173	0.000776
ENSLCOT00000000030.1	165	0.0011
ENSLCOT00000000015.1	3	0.000273
ENSLCOT00000000024.1	80	0.00221
ENSLCOT00000000032.1	80	0.00241
ENSLCOT00000000022.1	67	0.00271
ENSLCOT00000000039.1	13	0.000353
ENSLCOT00000000402.1	3	0.000125
ENSLCOT00000000353.1	10	0.000369
ENSLCOT00000000026.1	46	0.000303
ENSLCOT00000000023.1	5	0.000106
ENSLCOT00000000027.1	127	0.00384
ENSLCOT00000000029.1	10	0.000244
ENSLCOT00000000033.1	17	0.000672
ENSLCOT00000000045.1	4	0.000227
ENSLCOT00000000417.1	3	9.82e-05
ENSLCOT00000000452.1	3	9.18e-05
ENSLCOT00000000491.1	3	9.67e-05
ENSLCOT00000000080.1	42	0.000783
ENSLCOT00000000037.1	28	0.000925
ENSLCOT00000000036.1	11	0.000356
ENSLCOT00000000042.1	4	4.88e-05
ENSLCOT00000000093.1	25	0.00033
ENSLCOT00000000103.1	6	0.000151
ENSLCOT00000000249.1	13	0.000278
ENSLCOT00000000259.1	13	0.000255
ENSLCOT00000000260.1	23	0.00041
