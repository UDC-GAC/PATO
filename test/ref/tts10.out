# Duplex-ID	Start	End	Score	Strand	Error-rate	Errors	Guanine-rate	Duplicates	TTS	Duplicate locations
ENSLCOT00000000041.1	7990	8010	19	+	0.05	d3	0.45	2	GGAcAGAAGGAGAAAAGAGG	-
ENSLCOT00000000044.1	7711	7731	19	+	0.05	d3	0.45	2	GGAcAGAAGGAGAAAAGAGG	-
ENSLCOT00000000056.1	7435	7455	19	+	0.05	d3	0.45	2	GGAcAGAAGGAGAAAAGAGG	-
ENSLCOT00000000007.1	924	944	19	+	0.05	d2	0.6	0	AGcGAGGAGGAGAGGAAGGG	-
ENSLCOT00000000007.1	986	1002	16	+	0		0.56	0	GAAGGAGGAAGAGGAG	-
ENSLCOT00000000007.1	1173	1196	22	+	0.043	d18	0.57	0	GAGGAGGAAGAGGAGGAGcAGGA	-
ENSLCOT00000000007.1	1866	1887	20	+	0.048	d8	0.62	0	GGGGAGGAcGAAGAGGAGGGA	-
ENSLCOT00000000007.1	463	491	24	-	0.14	d2d26	0.57	0	AcAGAGGAGGAAGAGGAGGAGGGGGcAG	-
ENSLCOT00000000445.1	2732	2757	20	+	0.2	d2d21	0.48	2	GGcGAAGAAAGAGAGGGAAAGcAGG	-
ENSLCOT00000000484.1	2654	2679	20	+	0.2	d2d21	0.48	2	GGcGAAGAAAGAGAGGGAAAGcAGG	-
ENSLCOT00000000489.1	2651	2676	20	+	0.2	d2d21	0.48	2	GGcGAAGAAAGAGAGGGAAAGcAGG	-
ENSLCOT00000000017.1	611	632	20	+	0.048	d10	0.67	0	GGGGGGGAAGtGGAAGGAGAG	-
ENSLCOT00000000881.1	751	771	19	-	0.05	d8	0.5	2	AGAGGGAAGGGcAAAGAGAG	-
ENSLCOT00000000921.1	568	588	19	-	0.05	d8	0.5	2	AGAGGGAAGGGcAAAGAGAG	-
ENSLCOT00000000946.1	568	588	19	-	0.05	d8	0.5	2	AGAGGGAAGGGcAAAGAGAG	-
ENSLCOT00000000013.1	954	972	18	-	0		0.28	0	GAAAGAAAAAAGGAGAAA	-
ENSLCOT00000000012.1	3103	3124	20	-	0.048	d9	0.62	0	GGGGGAGAGAGtGAAGAGGGA	-
ENSLCOT00000000131.1	11	29	18	-	0		0.5	0	GAGAAGGAGAGGGAAAAG	-
ENSLCOT00000000069.1	1495	1513	18	+	0		0.56	0	GGGAGGAAGAGGAAGAGA	-
ENSLCOT00000000043.1	393	418	24	+	0.04	d6	0.44	0	GGGAAAtGGGGAAAAGAAGAGAAGA	-
ENSLCOT00000000043.1	483	508	24	+	0.04	d2	0.48	0	AGtAGAGAAAAGGAGGGGAGAAAGG	-
ENSLCOT00000000323.1	2151	2172	20	+	0.048	d15	0.71	1	GGGGAGGAGGAAGGGcGGAGG	-
ENSLCOT00000000346.1	1866	1887	20	+	0.048	d15	0.71	1	GGGGAGGAGGAAGGGcGGAGG	-
ENSLCOT00000000014.1	231	257	25	+	0.038	d21	0.38	0	AAAGAAAAGGGGAAGAGGAGAtAAAG	-
ENSLCOT00000000014.1	270	295	24	+	0.04	d9	0.4	0	AAAGAAGAAcGAGAAGGAGAGGGAA	-
ENSLCOT00000000014.1	1424	1446	21	+	0.045	d14	0.55	0	GAGAGGGAAAGAAAtGGGGAGG	-
ENSLCOT00000000014.1	3593	3614	20	+	0.048	d13	0.57	0	GAGGGAGGGAAAAtGGGGGAA	-
ENSLCOT00000000014.1	773	797	23	-	0.042	d21	0.5	0	AGcAGGAGAGGAGGAGGAAGAAGA	-
ENSLCOT00000000251.1	1298	1319	20	+	0.048	d8	0.38	1	GAAGGAAGcAGAGAAAGGAAA	-
ENSLCOT00000000251.1	4292	4308	16	+	0		0.44	1	GGAGAAAAGAGAAGGA	-
ENSLCOT00000000251.1	5031	5053	21	+	0.045	d7	0.27	1	GAGGAAGcAAAGAGAAAAAAAA	-
ENSLCOT00000000251.1	6533	6558	24	+	0.04	d20	0.48	1	GAGAGGAGAAGAGGAAGAAGtGGAA	-
ENSLCOT00000000251.1	7805	7826	20	+	0.048	d4	0.24	1	AGAAcAAAGAGAAGAAGAAAA	-
ENSLCOT00000000251.1	1989	2010	20	-	0.048	d18	0.43	1	AGtGGAAAAGGAGGGAAAAGA	-
ENSLCOT00000000255.1	1130	1151	20	+	0.048	d8	0.38	1	GAAGGAAGcAGAGAAAGGAAA	-
ENSLCOT00000000255.1	4124	4140	16	+	0		0.44	1	GGAGAAAAGAGAAGGA	-
ENSLCOT00000000255.1	4863	4885	21	+	0.045	d7	0.27	1	GAGGAAGcAAAGAGAAAAAAAA	-
ENSLCOT00000000255.1	6365	6390	24	+	0.04	d20	0.48	1	GAGAGGAGAAGAGGAAGAAGtGGAA	-
ENSLCOT00000000255.1	7637	7658	20	+	0.048	d4	0.24	1	AGAAcAAAGAGAAGAAGAAAA	-
ENSLCOT00000000255.1	1821	1842	20	-	0.048	d18	0.43	1	AGtGGAAAAGGAGGGAAAAGA	-
ENSLCOT00000000030.1	3971	3991	19	-	0.05	d14	0.45	0	GGGAGtAAAGGAGAAGAAAG	-
ENSLCOT00000000030.1	4873	4899	26	-	0		0.31	0	AAAAAGGAGAAGGGAAAAAAAAAGAG	-
ENSLCOT00000000024.1	831	849	18	+	0		0.22	1	AGAAGAAGAAAGAAAAAA	-
ENSLCOT00000000024.1	1071	1091	19	+	0.05	d5	0.3	1	GAAAAtGAAGAAAGAAAAGG	-
ENSLCOT00000000024.1	1377	1400	22	+	0.043	d3	0.3	1	AGAtGAAGAAGAAAAAGGAAAGA	-
ENSLCOT00000000032.1	831	849	18	+	0		0.22	1	AGAAGAAGAAAGAAAAAA	-
ENSLCOT00000000032.1	945	965	19	+	0.05	d5	0.3	1	GAAAAtGAAGAAAGAAAAGG	-
ENSLCOT00000000032.1	1251	1274	22	+	0.043	d3	0.3	1	AGAtGAAGAAGAAAAAGGAAAGA	-
ENSLCOT00000000022.1	5	35	26	+	0.13	d8d27	0.47	0	AAGAAGAGtGAAAAGGGGAGAGGAGGGcAA	-
ENSLCOT00000000039.1	56	77	20	-	0.048	d15	0.33	0	AAGAGcAGAAGAAAGAGAAGA	-
ENSLCOT00000000026.1	6307	6330	22	+	0.043	d5	0.48	0	GGGGAcGGAGGGAAGAAGAAAAA	-
ENSLCOT00000000027.1	486	505	19	+	0		0.37	0	AAAAGAAAAAGGGGAGAAG	-
ENSLCOT00000000027.1	22	47	24	-	0.04	d23	0.64	0	GcGGGAGGAGGGAAGGAAGGAGGGA	-
ENSLCOT00000000033.1	1026	1046	19	-	0.05	d14	0.4	0	GGAGGtGAAAAGGAAAAAAG	-
ENSLCOT00000000080.1	1639	1664	24	+	0.04	d16	0.52	0	GAGGGAAGAAGGAGAGcAAGGAGGA	-
ENSLCOT00000000093.1	3258	3280	21	+	0.045	d11	0.68	0	GGGGAGGGGGGtGAGAGGGAAA	-
ENSLCOT00000000093.1	1302	1323	20	-	0.048	d13	0.62	1	GGGGAAGtAGAGGAGGAGGGA	-
ENSLCOT00000000103.1	1311	1332	20	-	0.048	d13	0.62	1	GGGGAAGtAGAGGAGGAGGGA	-
ENSLCOT00000023191.1	121	147	25	+	0.038	d12	0.65	0	GGAGGAAGAGGAtGGAGAGGGGGAGG	-
ENSLCOT00000023191.1	1317	1334	17	-	0		0.59	0	GGGGAGGAAAAAGGGGA	-
ENSLCOT00000010487.1	0	23	22	+	0.043	d12	0.57	0	GAGGAAGGGAGAcGGGAGGAAAG	-
ENSLCOT00000010487.1	2494	2514	19	-	0.05	d5	0.7	0	GAGAGGGAGGAGGGcAGGGG	-
ENSLCOT00000010487.1	2807	2827	19	-	0.05	d10	0.75	0	GGGGGAAGGcGGGGAGGAGG	-
ENSLCOT00000010487.1	6086	6123	24	-	0.35	d11d34	0.59	0	AGtGAGGGGAGGGAAAAGGGGAAGAtGGGAGAGGAGG	-
ENSLCOT00000008897.1	4143	4159	16	+	0		0.5	0	GGAAGAGAGAGAAGAG	-
ENSLCOT00000008897.1	4310	4326	16	+	0		0.62	0	GGAGGAGGAAGGAGGA	-
ENSLCOT00000008897.1	5304	5326	21	+	0.045	d4	0.55	0	GGGGtGGGAAAAGAGAAGGGAA	-
ENSLCOT00000008897.1	5678	5694	16	+	0		0.69	0	GGAGGGGAGGGAAGAG	-
ENSLCOT00000008897.1	3177	3195	18	-	0		0.61	0	GAGAGGGGGAGAGAAAGG	-
ENSLCOT00000008897.1	3475	3498	22	-	0.043	d6	0.61	0	GGAGGAAGGAGGGGGAtGAAGGA	-
ENSLCOT00000008897.1	3662	3692	30	-	0		0.27	0	AAAGGAGGGGAAAAAAAAGAAAAAAAAAGA	-
ENSLCOT00000008897.1	4250	4291	29	-	0.29	d11	0.32	0	AGGGAAAAAAAGGGGGAAAAAAGAAAAGAcAAAAAAAGAGG	-
ENSLCOT00000004026.1	0	230	30	-	0.87	d38d105d159	0.57	0	GGGAAAGAGGGAGAAGGGGAGGGAAAGGGGAGAAGGAAAGGGGAGGGAAAGGAGGAGAAGAGGAGGGAAAtGGGAGGGAAAGAGGAGAGAAAAGGGAGAAGGAAAGAGGGAGAAGAAAAGAGGAtGGAGAGGGGAAGAAGGGGAGGGAAAGGGGGAGAAGGAGAGGGGAGGGAAGAGGAGGGAAAGGGGAGcGAAAGAGGAGGGAAAGGGGAGGGGGAGAGGAGGGAAAG	-
ENSLCOT00000004230.1	392	412	19	+	0.05	d13	0.75	0	GGAAAGGAGGGGGtGGGGGG	-
ENSLCOT00000004230.1	634	676	23	-	0.45	d17d38	0.52	0	GGGtGGAAGAGGAAGAGGAAGAGAtGAGGAGGAAGAAGAAGG	-
ENSLCOT00000004230.1	453	631	29	-	0.84	d160	0.5	0	GAGAAGGAGGAGGAAGAcAAGGAGAAGGAGAAGGAGAAGAAGGAGAAGGAGAAGGAGAAGGAGAAGGAGGAGGAGAAGGAGAAGGAGAAGGAGAAGGAGAAGAAGGAGAAGGAGGAGAAGGAGAAGGAGAAGGAGAAGGAGAAGGAGAAGAAGGAGAAGGAGAAGGAGAAGGAGAAGG	-
ENSLCOT00000019763.1	2661	2679	18	+	0		0.39	2	AAAAAGAAGGAGAAGGAG	-
ENSLCOT00000019764.1	2829	2847	18	+	0		0.39	2	AAAAAGAAGGAGAAGGAG	-
ENSLCOT00000019764.1	2311	2337	25	-	0.038	d24	0.54	1	AtAGAGGGGAAAGGGGAGAGGAGGAA	-
ENSLCOT00000019765.1	2775	2793	18	+	0		0.39	2	AAAAAGAAGGAGAAGGAG	-
ENSLCOT00000019765.1	2257	2283	25	-	0.038	d24	0.54	1	AtAGAGGGGAAAGGGGAGAGGAGGAA	-
